app/facades/all_recipes_scrape_facade.rb

```diff
@@ -16,8 +16,8 @@ def recipe_details(url)
       instructions_list << single
     end
     
-    name = document.css("h1.comp").first.children.text.gsub(/\n/,"") 
-    joined_instructions = instructions_list
+    name = document.css("h1.article-heading").first.children.text.gsub(/\n/,"") 
+    joined_instructions = instructions_list.join
```

Comment:
I think I removed the join because FE wanted the instructions in an array

Reply:
Oh my bad